src/components/Input/Input.js

```diff
@@ -331,7 +334,7 @@ const StyledInput = ({
             invalid,
             disabled,
             hasWarning,
-            ref: deepRef,
+            ref: ref || deepRef,
```

Comment:
Feel free to remove the `deepRef`. We'll provide a codemod for it.

Reply:
Can you please rename it [here](https://github.com/sumup-oss/circuit-ui/blob/alpha/src/components/CurrencyInput/CurrencyInput.js#L104) as well?